services/spar/src/Spar/App.hs

```diff
@@ -72,12 +73,15 @@ instance HasCreateUUID Spar where
 instance HasLogger Spar where
   -- FUTUREWORK: optionally use 'field' to index user or idp ids for easier logfile processing.
   logger (toLevel -> lv) mg = do
+    let mg'    = Log.msg . condenseLogMsg . cs $ mg
+    log lv mg'
```

Comment:
This is correct, but while you're at it, could you:

```haskell
  logger lv = log (toLevel lv) . Log.msg . cs
```

`condenseLogMsg` should be deleted.  That's from an older version of tinylog that didn't handle newlines in log messages.